package.json

```diff
@@ -32,6 +32,7 @@
     "chance": "^1.0.18",
     "cheke": "^1.0.3",
     "cors": "^2.8.5",
+    "d": "^1.0.0",
```

Comment:
what's this meant for?

Reply:
The package is no longer needed, I am removing it